README.md

```diff
@@ -1,59 +1,111 @@
-<p align="center">
-	<img src="https://user-images.githubusercontent.com/30529572/72455010-fb38d400-37e7-11ea-9c1e-8cdeb5f5906e.png" />
-	<h2 align="center"> VIT Events - Backend </h2>
-	<h4 align="center"> The project consists of modules that together work to fetch the Daily Events Email sent by the students welfare office, then extract the various events happening on a particular day and then add it to once's Google Calender. <h4>
+
+<p  align="center">
+
+<img  src="https://user-images.githubusercontent.com/30529572/72455010-fb38d400-37e7-11ea-9c1e-8cdeb5f5906e.png"  />
+
+<h2  align="center"> VIT Events - Backend </h2>
+
+<h4  align="center"> The project consists of modules that extract information from VIT Student Welfare Department's Daily Events Emails. <h4>
+
 </p>
 
+  
+
 ---
-[![DOCS](https://img.shields.io/badge/Documentation-see%20docs-green?style=flat-square&logo=appveyor)](INSERT_LINK_FOR_DOCS_HERE) 
-  [![UI ](https://img.shields.io/badge/User%20Interface-Link%20to%20UI-orange?style=flat-square&logo=appveyor)](INSERT_UI_LINK_HERE)
+
+[![DOCS](https://img.shields.io/badge/Documentation-see%20docs-green?style=flat-square&logo=appveyor)](INSERT_LINK_FOR_DOCS_HERE)
+
 
 
+  
+  
+
 ## Functionalities
-- [ ] Fetches the emails from VIT Student Mailbox.
-- [ ] Extracts the information about various events attached with the mail.
-- [ ] Adds the events to Google Calender.
+
+- [X] API to crop image to extract informatin
+- [X]  API to segment data to proper rows
+  
 
 <br>
 
+  
+  
 
 ## Instructions to run
 
+  
+
 * Pre-requisites:
-	-  googleapiclient
-	-  google_auth_oauthlib
-	-  base64
-	-  email
-	-  apiclient
-	-  PIL
-	-  pickle
-	-  numpy
-	-  opencv
-	
+
+- googleapiclient
+
+- google_auth_oauthlib
+
+- base64
+
+- email
+
+- apiclient
+
+- PIL
+
+- pickle
+
+- numpy
+
+- opencv
+
+  
 
 * To satisfy the requirements run the command:
+
 ```bash
+
 sudo pip install -r requirements.txt
+
 ```
 
-* Command to execute on bash terminal:
+  <br/>
+
+* Command to publish on heroku
+
+  
 
 ```bash
-python main.py
+heroku login
+git init
+heroku create
+git add .
+git commit -m "<YOUR MESSAGE>"
+git push heroku master
 ```
+* API endpoints
+		1.  /crop
+		2. /cropb64
+		3. /getTable
 
```

Comment:
NO need to write it here. Prepare the docs and add link in the readme in the badge as given in the template.

Reply:
I'll change it once the docs are done.